### ISSUE TITLE
lacks ability to produced code for subpackages

### ISSUE THREAD

**Issue by arvenil** [davecheney/godoc2md#6]
```
$ godoc2md github.com/uniplaces/uniplaces-ap-api-client/...
2016/01/29 16:18:08 open /usr/local/go/src/github.com/uniplaces/uniplaces-ap-api-client/...: no such file or directory
```

**Comment by lordnynex** [comment]
@arvenil This is not a bug of godoc2md. This is the way godoc works. You should look into writing a wrapper around godoc2md to recursively walk github.com/a/b/. Something like this:

```bash
PACKAGES="$(go list -f "{{.Name}}:{{.ImportPath}}" ./... | grep -v -E "main:|vendor/" | cut -d ":" -f 2)"; \
	for PACKAGE in $PACKAGES; do \
		PACKAGE_NAME="$(go list -f "{{.Name}}" $PACKAGE)"; \
		mkdir -p $PACKAGE_NAME/; \
	  godoc2md $PACKAGE > $PACKAGE_NAME/README.md; \
	done
```

**Comment by erizocosmico** [comment]
@lordnynex it does not produce a `Subdirectories` section, though, despite having it on the upper menu.

**Comment by davecheney** [comment]
That's true, I stripped out that part of the template when I converted it
to markdown. However this would imply that you run godoc2md in each package
in your project.

While this is something you could do, the goal of godoc2md was really to
put _something_ in the readme at the root of a project without having to
duplicate the documentation you'd already entered in godoc format.

**Comment by erizocosmico** [comment]
@davecheney fair point. Having that feature is not very well suited for a README. I opened #12 before reading your reply so feel free to close it, since it does not make much sense. I can use a custom template for my specific needs :P

Do you want me to open another PR to remove the `Subdirectories` link from the upper menu, then? If the table with the subpackages is not gonna be present there's no point in keeping the link.
